docs/get-started/lotus/installation.md

```diff
@@ -363,23 +397,63 @@ We recommend waiting until the syncing process has completed, which should be re
 lotus sync wait
 ```
 
+While waiting the syncing process, you can also check  status of the syncing process.
+
+```sh
+lotus sync status
+```
+
 ## Interact with the daemon
 
 The `lotus` command allows you to interact with a _running_ Lotus daemon. The `lotus-miner` and `lotus-worker` commands work in the same way.
 
-Lotus comes with built-in CLI documentation:
+Lotus comes with built-in CLI documentation. Based on different usages, lotus provides different sub-commands for developers. You can also use `help` to check the usage for any sub-command.
```

Comment:
```suggestion
Lotus comes with built-in CLI documentation.
```

These instructions are already really long, and you have nice examples below. I made some suggestions to incorporate this content inside lines 413-423